### PR TITLE
Class Factory Example

### PR DESCRIPTION
**Pull request by kjvalencik**

### CODE REVIEW

File: class-factory/native/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[package]
+name = "class-factory"
+version = "0.1.0"
+build = "build.rs"
+exclude = ["artifacts.json", "index.node"]
+
+[lib]
+name = "class_factory"
+crate-type = ["cdylib"]
```
Comment (amilajack) [inline class-factory/native/Cargo.toml:9]: Should we change all the `dylib`'s in the examples to `cdylib`?

Comment (kjvalencik) [reply class-factory/native/Cargo.toml:9]: Yes, it's a fix in the template that is part of `0.3.1`.

Comment (amilajack) [reply class-factory/native/Cargo.toml:9]: Nice! Should we make those changes to the other examples in the repo in another PR or in this PR?

Comment (amilajack) [reply class-factory/native/Cargo.toml:9]: Actually I'll do this in a separate PR

**Pull request merged**